static/beta/stage/navigation/settings-navigation.json

```diff
@@ -68,7 +68,12 @@
                     "href": "/settings/notifications/configure-events",
                     "permissions": [
                       {
-                        "method": "isOrgAdmin"
+                        "method": "loosePermissions",
+                          "args": [
+                            "isOrgAdmin",
```

Comment:
This won't work, the `isOrgAdmin` should be another permission method. @Hyperkid123 do we have `or` between permission methods or `and`?
```suggestion
```

Reply:
We only have `and`

The permission `isOrgAdmin` does not exist. However, org admin should have the specific permission anyway. So finding using permission strings should work. I might be wrong.